.github/workflows/release.yaml

```
@@ -0,0 +1,98 @@
+          ## Cosmopilot v${VER}
+
+          **Container images**
+          - Multi-arch manifest: `${IMG}:${VER}@${MANIFEST_DIGEST}`
+          - linux/amd64: `${IMG}:${VER}-amd64@${AMD64_DIGEST}`
+          - linux/arm64: `${IMG}:${VER}-arm64@${ARM64_DIGEST}`
+
+          **Install / upgrade with Helm (from OCI)**
+          ```bash
+          helm install cosmopilot oci://ghcr.io/${GITHUB_REPOSITORY_OWNER,,}/helm/cosmopilot
+          ```
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)